swimster-api/middleware/multer.js

```diff
@@ -0,0 +1,27 @@
+const multer = require('multer')
+const path = require('path')
+const { BadRequestError } = require('../utils/error')
+
+// stores the files in memory as Buffer objects
+const storage = multer.memoryStorage()
+
+// controls which files are supported - jpg/jpeg
+const fileFilter = (req, file, cb) => {
+    if (file.mimetype == "image/jpeg" || file.mimetype == "image/jpg") {
+        return cb(null, true)
+    } else {
+        cb(null, false)
+        const error = new BadRequestError("Only .jpeg and .jpg are allowed!")
```

Comment:
`Only .jpeg and .jpg file types are allowed!`

Reply:
done!